packages/back-end/src/api/openapi/paths/postSavedGroup.yaml

```
@@ -0,0 +1,48 @@
+parameters: []
+tags:
+  - saved-groups
+summary: Create a single saved group
+operationId: postSavedGroup
+x-codeSamples:
+  - lang: "cURL"
+    source: |
+      curl -X POST https://api.growthbook.io/api/v1/saved-groups/grp_123abc \
+        -H "Content-Type: application/json" \
+        -u secret_abc123DEF456: \
+        -d '{"name": "interal-users", "values": "userId-123, userId-345, userId-678", "attributeKey": "userId"}'
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        type: object
+        required:
+          - name
+          - values
+          - attributeKey
+        properties:
+          name:
+            type: string
+            description: The display name of the Saved Group
+          values:
+            type: array
+            items:
+              type: string
+            description: "An array of values to target (Ex: a list of userId's)."
+          attributeKey:
+            type: string
+            description: "The parameter you want to target users with. Ex: userId, orgId, ..."
+          owner:
+            type: string
+            description: (Optional) - The name of the user in your GrowthBook account that owns this Saved Group.
```

Copy nit:

```suggestion
            description: The name of the user in your GrowthBook account that owns this Saved Group.
```

When we provide a field under the required fields, it'll show that it's required.